### PR TITLE
f-searchbox@v4.0.0-beta.10 - Update readme and fix numeric field styles.

### PR DESCRIPTION
**Pull request by kevinrodrigues**
### Added
- Readme improvements to the follow:
  - Events table.
  - Props table.

### Fixed
- Street number overlay issue when address is too long.
- Geo location icon postion when street number is visible.

### Before
Screen Shot 2020-12-10 at 10 15 20 AM

### After
Screen Shot 2020-12-10 at 10 13 55 AM

### CODE REVIEW

File: packages/f-searchbox/README.md [hunk 1]
```diff
@@ -94,13 +97,55 @@ const options = {                 // (opt) component settings
...
+### Props
+
+`f-searchbox` has a number of props that allow you to customise its functionality.
+
+The props that can be defined are as follows:
+
+| Prop                      | Required       | Type          | Default | Description |
+| :---                      |     :---:      |     :---:     |  :---:  | :---        |
+| locale                    | false          | `string`      | `en-GB` | Sets the locale of the component (which determines what theme and translations to use |
+| config                    | false          | `object`      | `{}`    | When passed in `f-searchbox` will allow presentational & functional customisation (see options section) |
```
Comment (DamianMullins) [inline packages/f-searchbox/README.md:120]: Probably worth adding some detail at some point on what this object can contain.

**Review by DamianMullins — DISMISSED** [review]

File: packages/f-searchbox/README.md (hunk 1 above)
Comment (kevinrodrigues) [reply packages/f-searchbox/README.md:120]: Do you mean add the `options` (further up) into this table?

Comment (DamianMullins) [reply packages/f-searchbox/README.md:120]: Ah, yes. Maybe that section can be updated at some point to be in the same table format?

Comment (DamianMullins) [reply packages/f-searchbox/README.md:120]: I think the fact the variable was labelled as "options" rather than "config" threw me.

Comment (kevinrodrigues) [reply packages/f-searchbox/README.md:120]: Yeah i just noticed that too. Ill update that.

Comment (kevinrodrigues) [reply packages/f-searchbox/README.md:120]: Fixed.

File: packages/f-searchbox/src/components/formElements/FormSearchField.vue [hunk 2]
```diff
@@ -82,7 +84,9 @@ export default {
 
     computed: {
         ...mapState('searchbox', [
-            'shouldInputFieldHaveFocus'
+            'shouldInputFieldHaveFocus',
+            'streetNumberRequired',
```
Comment (ashleynolan) [inline packages/f-searchbox/src/components/formElements/FormSearchField.vue:88]: `isStreetNumberRequired`?

**Review by ashleynolan — COMMENTED** [review]
Looks good - just one small comment

File: packages/f-searchbox/src/components/formElements/FormSearchField.vue (hunk 2 above)
Comment (kevinrodrigues) [reply packages/f-searchbox/src/components/formElements/FormSearchField.vue:88]: Mind if i add this to the next PR? Have already updated in the next one 👍

Comment (ashleynolan) [reply packages/f-searchbox/src/components/formElements/FormSearchField.vue:88]: Yep – that's cool

**Review by ashleynolan — APPROVED** [review]

**Review by simonsandersje — APPROVED** [review]

**Review by AlixRichards — APPROVED** [review]

File: packages/f-searchbox/src/components/formElements/FormSearchField.vue (hunk 2 above)
Comment (kevinrodrigues) [reply packages/f-searchbox/src/components/formElements/FormSearchField.vue:88]: Fixed: https://github.com/justeat/fozzie-components/pull/537

**Pull request merged**